content/blog/highly-available-nfs-workload-on-hpe-greenlake-for-private-cloud-enterprise-using-serviceguard-for-linux.md

```diff
@@ -0,0 +1,748 @@
+
+Set up the required parameters for hpegl provider that was specified earlier. As previously explained, you can either explicitly set those parameters in your TF file or have them set in a series of environment variables or have a mix of both. It is recommended to add the following two parameters in your TF file:
+
+```json
+# Setup provider environment (location and space)
+provider "hpegl" {
+      vmaas {
+         location   = "FTC06"
+         space_name = "Default"
+      }
+}
+```
+
+The rest (such as tenant id, user id and user secret key) can be placed in a RC file, which you can source before running your Terraform command.
+You can find your location and your space name from the HPE GreenLake for Private Cloud Enterprise Overview. In the example shown below, FTC06 is our location:
+
+![](/img/picture1.png)
+
+In the capture below, Default is the space you will use for your work with Terraform. You can check your available Spaces from the HPE GreenLake console under your profile icon, Change Space.
+
+![](/img/picture2.png)
+
+## Setting up API Client access
+
+Next, you need to create a new API Client access dedicated to Terraform. You can do this from the HPE GreenLake console under your settings icon, select User Management and then the API Clients tab.
+
+![](/img/picture3.png)
+
+Create a new API Client and be sure to note down the Issuer, Client ID and Client Secret values which are shown. 
+The value for the tenant ID may be seen in the Tenant ID field under the API Access menu and in the URL of your browser.
+
+![](/img/picture4.png)
+
+With this you can now build a resource file that defines the following environment variables:
+
+```shell
+export HPEGL_TENANT_ID=<Your Tenant ID>
+export HPEGL_USER_ID=<Client ID of the API Client>
+export HPEGL_USER_SECRET=<Secret Key displayed when you created the API Client>
+export HPEGL_IAM_SERVICE_URL=<Issuer URL>
+```
+
+And execute it on your machine to set these environment variables.
+
+## Assign Roles to API Client
+
+Once your API Client has been created, you need to assign a Role and a Space. You can assign a Role and a Space by clicking on your new API Client and then clicking the Create Assignment button. 
+Since intent is to use this API Client to create resources in the Virtual Machines Service, we need to assign an appropriate Virtual Machines Role. Choose a Role like ‘Private Cloud Tenant Contributor’ and choose the same Space as used earlier, I.e., ‘Default.’ 
+
+Note: More details on HPE GreenLake user roles can be found in the HPE GreenLake documentation.
+
+Set API Client Usernames and Passwords
+
+When a user creates virtual machines using the HPE GreenLake for Private Cloud Enterprise: Virtual Machines user interface, they first set the Linux and Windows username and password. Once this is done, any virtual machines subsequently created by that user will inherit these credentials. The user can later use these credentials to log into these virtual machines.
```

Comment:
If you look at the preview for this paragraph: https://deploy-preview-2148--hpe-dev-portal.netlify.app/blog/highly-available-nfs-workload-on-hpe-greenlake-for-private-cloud-enterprise-using-serviceguard-for-linux/
You may find helpful for the readers to separate some paragraph. 

Also in Markdown mode of the editor, use the escape character "a backslash" for the variables "VM\\_USERNAME", VM\\_PASSWORD

Not sure what you mean with "HPEGL** environment"  in the last sentence of this section. Do you want to highlight in Bold the word "HPEGL"?